git_tools/tag_all_repos.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+"""
+*****************************************************************
+Licensed Materials - Property of IBM
+(C) Copyright IBM Corp. 2020. All Rights Reserved.
+US Government Users Restricted Rights - Use, duplication or
+disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+*****************************************************************
+
+*******************************************************************************
+Script: tag_all_repos.py
+A script that can be used to create the same annotated git tag in all
```

Comment:
Can you add a short example of using this? Maybe the bare minimum of what you would need to pass to the script? It could either be here, or in a README file somewhere.

Reply:
I added the example here. What do you think?

Reply:
Yeah, it's still sorta in flux.